File: javascript/product.js
class Product {
    constructor(code, name, price) {
      this.code = code;
      this.name = name;
      this.price = price;
    }
    // Getter
    get get_code() {
      return this.code;
    }

    get get_name() {
        return this.name;
    }

    get get_price() {
        return this.price;
    }
    // Method
  }
  
  const product = new Product(123, 'Aerofólio para o meu Palio', 250);

File: dart/product.dart
class Product {
  late String _name;
  late String _code;
  late int _price;

  Product(String name, String code, int price) {
    this.code = code;
    this.name = name;
    this.price = price;
  }

  String get name => _name;
  set name(String name) => _name = name;

  String get code => _code;
  set code(String code) => _code = code;

  int get price => _price;
  set price(int price) => _price = price;
}
